projects/u-mill/white_mesa/decantation_filtration.py

```diff
@@ -116,27 +116,25 @@ def __init__(self):
         # Domain attributes
 
         # Configuration parameters
-        self.std_tank_volume = 3402.33 * unit.meter**3
-        self.wash_water_preleach_feed_ratio = 99.0
-        self.std_underflow_overflow_mass_flowrate_ratio = 99.0
-        self.wash_water_acidleach_feed_ratio = 1.69
-
+        self.std_tank_volume = 3402.33 * unit.meter**3 # STD
         self.ccd_tank_volume = 7*339.29 * unit.meter**3 # CCD
+        self.vol_drum = 30.2 * unit.meter**3 #Filtration
+
+        self.std_wash_water_vol_flowrate = 1067 * unit.gallon/unit.minute
         self.ccd_wash_water_vol_flowrate = 1067 * unit.gallon/unit.minute
+
+        self.wash_water_preleach_feed_ratio = 99.0
+        self.std_underflow_overflow_mass_flowrate_ratio = 99.0
         self.ccd_underflow_overflow_mass_flowrate_ratio = 99.0
+        self.wash_water_acidleach_feed_ratio = 1.69
 
         # Initialization
         self.wash_water_mass_density = 1.0 * unit.kg/unit.meter**3
 
         # Decantation
-        if self.get_port('std-feed').connected_port:
```

Comment:
Why are removing this if block?

Reply:
This must have reverted back by accident. I had to redownload the entire repository due to the extent of conflicts in my version of the code. This can be put back.